e2e_report_test.go

```diff
@@ -75,3 +77,116 @@ func TestE2EReport_Aliases(t *testing.T) {
 
 	assert.Contains(t, rep.reported, "foo.Object().ContainsKey()")
 }
+
+const (
+	intSize = 32 << (^uint(0) >> 63) // 32 or 64
+	maxInt  = 1<<(intSize-1) - 1
+)
```

Comment:
Can we just use, say, 1000000? I guess there is no real need for using max int here?